app/models/orchestration_template.rb

```diff
@@ -14,11 +14,14 @@ class OrchestrationTemplate < ApplicationRecord
 
   validates :md5,
             :uniqueness => {:scope => :draft, :message => "of content already exists (content must be unique)"},
-            :unless     => :draft?
+            :if         => :unique_md5?
   validates_presence_of :name
 
   before_destroy :check_not_in_use
 
+  attr_accessor :remote_proxy
```

Comment:
remote_proxy is there so we can activate this behavior only for UI Forms, but not for refresh